specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json

```diff
@@ -272,6 +272,75 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.OperationalInsights/deletedWorkspaces": {
```

Comment:
>deletedWorkspaces [](start = 77, length = 17)

Is this going to be a proxy or tracked resource?

Reply:
It's a proxy resource that references a recently deleted tracked resource. When a user takes such resource and do a PUT on it, it will be recovered.